packaging/windows/khiops_env.cmd

```diff
@@ -0,0 +1,158 @@
+echo.
+echo The following variables are not defined by default and can be used to change some default
+echo  properties of Khiops:
+echo.
+echo KHIOPS_TMP_DIR: Khiops temporary directory location (default: the system default).
+echo   This location can be modified from the tool as well.
+echo KHIOPS_MEMORY_LIMIT: Khiops memory limit in MB (default: the system memory limit).
+echo   The minimum value is 100 MB; this setting is ignored if it is above the system's memory limit.
+echo   It can only be reduced from the tool.
+echo KHIOPS_API_MODE: standard or api mode for the management of output result files created by Khiops
+echo   In standard mode, the result files are stored in the train database directory,
+echo   unless an absolute path is specified, and the file extension is forced if necessary.
+echo   In api mode, the result files are stored in the current working directory, using the specified results as is.
+echo   . default behavior if not set: standard mode
+echo   . set to 'true' to force standard mode
+echo   . set to 'false' to force api mode
+echo KHIOPS_RAW_GUI: graphical user interface for file name selection
+echo   . default behavior if not set: depending on the file drivers available for Khiops
+echo   . set to 'true' to allow file name selection with uri schemas
+echo   . set to 'false' to allow local file name selection only with a file selection dialog
+echo.
+echo In case of configuration problems, the variables KHIOPS_JAVA_ERROR and KHIOPS_MPI_ERROR contain error messages.
+
+if not %2.==. exit /b 1
+if %1.==-h. exit /b 0
+if %1.==--help. exit /b 0
+exit /b 1
+
+REM Set Khiops environment variables
+:DISPLAY_ENV
+setlocal
+set DISPLAY_ENV=true
+
+:SET_ENV
+REM Initialize exported variables
+set "KHIOPS_PATH="
+set "KHIOPS_COCLUSTERING_PATH="
+set "KHIOPS_MPI_COMMAND="
+set "KHIOPS_JAVA_PATH="
+set "KHIOPS_CLASSPATH="
+set "KHIOPS_JAVA_ERROR="
+set "KHIOPS_MPI_ERROR="
+
+REM Set Khiops home to parent directory
+for %%a in ("%~dp0..") do set "_KHIOPS_HOME=%%~fa"
+
+REM KHIOPS_PATH
+set "KHIOPS_PATH=%_KHIOPS_HOME%\bin\MODL.exe"
+set "KHIOPS_COCLUSTERING_PATH=%_KHIOPS_HOME%\bin\MODL_Coclustering.exe"
+
+REM KHIOPS_LAST_RUN_DIR
+if "%KHIOPS_LAST_RUN_DIR%".=="". set "KHIOPS_LAST_RUN_DIR=%USERPROFILE%\khiops_data\lastrun"
+
+REM KHIOPS_PROC_NUMBER
+if "%KHIOPS_PROC_NUMBER%".=="". for /f %%i in ('"%~dp0_khiopsgetprocnumber"') do set "KHIOPS_PROC_NUMBER=%%i"
+if "%KHIOPS_PROC_NUMBER%".=="". set "KHIOPS_PROC_NUMBER=1"
+
+REM set MPI environment
+if %KHIOPS_PROC_NUMBER% LEQ 2 goto MPI_DONE
+REM ... with the standard variable MSMPI_BIN
+set "KHIOPS_MPI_COMMAND=%MSMPI_BIN%mpiexec.exe"
+REM ... if MSMPI_BIN is not correctly defined 
+REM ... we try to call directly mpiexec (assuming its path is in the 'path' variable)
+if not exist "%KHIOPS_MPI_COMMAND%" goto USE_MPI_IN_PATH
+:MPI_PARAMS
+REM ... we add the MPI parameters
+if not "%KHIOPS_MPI_COMMAND%."=="." set "KHIOPS_MPI_COMMAND="%KHIOPS_MPI_COMMAND%" -n %KHIOPS_PROC_NUMBER%"
```

Comment:
In the commit message of commit https://github.com/KhiopsML/khiops/pull/343/commits/b37dbbf98eb7867ba7707c62342eec463e97d38d where this change is introduced (process affinity removal), fix typo:
s/remove the prioryty/remove the priority/